src/api/client/resolvers.ts

```diff
@@ -298,14 +298,14 @@ export const queryNodeStitchingResolvers = (
         info
       )
 
-      const followsLookup = createLookup<{ id: string; views: number }>(batchedChannelFollows || [])
+      const followsLookup = createLookup<{ id: string; follows: number }>(batchedChannelFollows || [])
```

Comment:
This change should be done via rebase of `video-content-pages` right?

Reply:
I see. If you rebase `video-content-pages` with `dev` and then your branch with `video-content-pages` it should go away